buildlib/tests.yml

```diff
@@ -35,6 +25,8 @@ jobs:
           worker: $(worker_id)
           BUILD_NUMBER: "$(Build.BuildId)-$(Build.BuildNumber)"
           JOB_URL: "$(System.TeamFoundationCollectionUri)$(System.TeamProject)/_build/results?buildId=$(Build.BuildId)"
-          EXECUTOR_NUMBER: $(executor_number)
-          JENKINS_RUN_TESTS: 1
+          JENKINS_RUN_TESTS: ""
+          EXECUTOR_NUMBER: $(AZP_AGENT_ID)
```

Comment:
who sets AZP_AGENT_ID?

Reply:
Set for each new agent while configuring it. Uniq for one host.

Reply:
So you set it manually to 0,1,2,.. etc?

Reply:
Yes, starting with 1. The CPU affinity is set from 0.

Reply:
10x can you pls add code comment in here to explain?

Reply:
comment: from 1 to N